distribution/docs/src/main/jdocs/templates/documentation.ftl

```diff
@@ -0,0 +1,31 @@
+= ${branding-expanded} Documentation
+include::${project.build.directory}/doc-contents/_contents/jconfig.adoc[]
+
+
+:sectnums!:
+== Introduction
+:sectnums:
```

Comment:
Potentially, these major sections could be separated into their own templates, which would clean up what appears at this top-level.